packages/ui-library/src/components/token-picker/index.tsx

```diff
@@ -176,8 +176,8 @@ const Row: ItemContent<TokenWithBalance, RowData> = (
     <StyledForegroundPaper key={`${token.chainId}-${token.address}`} onClick={() => onClick(tokenWithBalance)}>
       <ContainerBox flex="1" alignItems="center" gap={3}>
         {token.icon}
-        <ContainerBox flexDirection="column" flex="1" alignItems="flex-start">
-          <Typography variant="bodyBold" color={colors[themeMode].typography.typo2}>
+        <ContainerBox flexDirection="column" flex="1" alignItems="flex-start" gap={0.5}>
+          <Typography variant="bodyBold" color={colors[themeMode].typography.typo2} lineHeight={1}>
```

Comment:
this has a specific different line height?

Reply:
By default it's line height is 2, so in order to make the line thiner I made that change. With default line height:
<img width="521" alt="Screenshot 2024-09-30 at 1 54 42 PM" src="https://github.com/user-attachments/assets/153fd11c-87c2-418d-86b6-d1bb69190c8e">